doc/source/user/build-images.rst

```diff
@@ -0,0 +1,24 @@
+=================================
+Build and consume your own images
+=================================
+
+If you want to build and consume your own image, for example, for libvirt, set
+the following in your `${WORKDIR}/env/extravars`:
+
+
+.. code-block:: yaml
+
+   ---
+   myregistry: "jevrard-osh.openstack.local:5000/"
+   developer_mode: "True"
+   # Builds the libvirt image from OSH-images repository.
+   docker_images:
+     - context: libvirt
```

Comment:
It might be useful to specify custom Dockerfile here as well

Reply:
Good idea.